css/intro.css

```diff
@@ -86,6 +86,10 @@
 }
 
 @media screen and (max-width: 500px) {
+  .input-label-dark {
+    font-size: 0.9rem;
```

Comment:
NitPick: 
Using random values like these makes the design feel inconsistent. If even we do not have a design system in place please use multiples of 4 (4, 8, 16, 20, 24...) converted into rem units (0.25rem, 0.5rem, 0.75rem ...) for font sizes and spacing units (padding / margin) as it will make the layout look neat and consistent.

Reply:
Thank you @yesyash for this valuable feedback